nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -711,7 +711,7 @@ Use `services.pipewire.extraConfig` or `services.pipewire.configPackages` for Pi
   and `services.kavita.settings.IpAddresses`. The file at `services.kavita.tokenKeyFile` now needs to contain a secret with
   512+ bits instead of 128+ bits.
 
-- `services.netbird` now allows running multiple tunnels in parallel through [`services.netbird.tunnels`](#opt-services.netbird.tunnels).
+- `services.netbird` now allows running multiple tunnels in parallel through [`services.netbird.tunnels`](#opt-services.netbird.clients).
```

Comment:
to be honest, this PR is hanging for so many months I lost hope it will ever get merged and I stopped trying to chase people able to merge it.

Reply:
I'll make the "proper" realease notes when some of the maintainers able to give it review and merge pops up.
The code itself was written around 23.11 and already missed 2 releases.

Reply:
I know how frustrating a long running PR can be. The unfortunate thing is that any committer who comes along will see that this is missing proper release notes, and then can't merge it. So this will only delay it further.

I fully understand if you say that this requires too much work to keep up in the long term. Let me know if you need support with anything, otherwise I'd also offer pick this PR up if you don't want to keep working on it.